packages/coil-extension/src/types/user.ts

```diff
@@ -1,7 +1,10 @@
 export interface User {
+  newUi?: boolean // todo: remove this when testing the new ui is done
```

Comment:
Is the effectively a feature flag? Maybe we should call this like `newUi2022` or `newUiV2` so we have more context. `newUiTipping`

Reply:
There's no real data behind it or db entry for it. It's just a variable I flip when working on it, i'll be the only one using it. Once the UI is updated everyone will get the new UI and the tipping will still be behind the `tipping-beta` feature flag.

Reply:
kk, that makes sense to just leave it then. 👍